packages/react-tabster/src/hooks/useFocusIndicatorStyle.ts

```diff
@@ -21,7 +22,7 @@ export type FocusOutlineStyleOptions = {
  * @param options - Configures the style of the focus outline
  * @returns focus outline styles object
  */
-const getFocusOutlineStyles = (options: FocusOutlineStyleOptions) => {
+const getFocusOutlineStyles = (options: FocusOutlineStyleOptions): GriffelStyle => {
```

Comment:
This forcefully enables type-checking that didn't happen before for unknown reason.